index.js

```
@@ -18,7 +18,7 @@ const apodNasa = () => {
 			image: `https://apod.nasa.gov/apod/image${link}`
 		};
 	});
-};
+}.catch(e => console.log(e));
```

The `catch` parameter should be added at line __[`20`](https://github.com/CodeDotJS/apod-nasa/blob/master/index.js#L20)__, not __`21`__, and you can return a sensible error like ~
```js
    catch(err => {
        if (err) {
            err.message = 'Content is unavilable or is in unsupported format!'
        }
    return err.message;
});
```